api/batches/batch.go

```diff
@@ -20,6 +20,7 @@ const jobBasePath = "/job-batches-api/v2/projects/"
 type Batch interface {
 	Create(ctx context.Context, projectID string, payload CreateBatchPayload) (CreateBatchResponse, error)
 	CreateJob(ctx context.Context, projectID string, payload CreateJobPayload) (CreateJobResponse, error)
+	GetJob(projectID string, translationJobUID string) (GetJobResponse, error)
```

Comment:
You added `GetJob` into the wrong class. We have two APIs: Jobs API; Job Batches. You need to create a new class specifically for Jobs API. https://api-reference.smartling.com/#tag/Jobs